modules/general.rb

```diff
@@ -91,6 +91,13 @@ module GeneralCommands
       command(:lug, aliases: %i[plug]) do
         'https://discord.gg/3Jfq6aXy5B 🔌'
       end
+
+      command(:roll) do |event, n|
+        if n.to_i <= 0
+          "You need to put in an integer that is equal to or greater than 1."
+        else
```

Comment:
lets make this simpler and have the bot react with :question: if its not an int or not >1

Reply:
Fixed

Reply:
I mean like `event.react "?"` _a la_ `!role`

Reply:
Ah, will re-fix XD